app/lib/pdf-generator.ts

```diff
@@ -0,0 +1,94 @@
+import { PDFDocument, rgb, StandardFonts } from 'pdf-lib';
+
+export interface ContractData {
+  employeeName?: string;
+  position?: string;
+  startDate?: string;
+  salary?: string;
+  companyName?: string;
+}
+
+export async function generateHiringAgreementPDF(data: ContractData = {}): Promise<Uint8Array> {
+  const pdfDoc = await PDFDocument.create();
+  const page = pdfDoc.addPage([612, 792]); // Standard letter size
+  const font = await pdfDoc.embedFont(StandardFonts.Helvetica);
+  const boldFont = await pdfDoc.embedFont(StandardFonts.HelveticaBold);
+  
+  const { width, height } = page.getSize();
+  const margin = 50;
+  let yPosition = height - margin;
+  
+  const addText = (text: string, fontSize: number = 12, fontType = font, color = rgb(0, 0, 0)) => {
+    page.drawText(text, {
+      x: margin,
+      y: yPosition,
+      size: fontSize,
+      font: fontType,
+      color,
+    });
+    yPosition -= fontSize + 8;
+  };
+  
+  const addParagraph = (text: string, fontSize: number = 12, fontType = font) => {
+    const words = text.split(' ');
+    let line = '';
+    const maxWidth = width - (margin * 2);
+    
+    for (const word of words) {
+      const testLine = line + (line ? ' ' : '') + word;
+      const textWidth = fontType.widthOfTextAtSize(testLine, fontSize);
+      
+      if (textWidth > maxWidth && line) {
+        addText(line, fontSize, fontType);
+        line = word;
+      } else {
+        line = testLine;
+      }
+    }
+    
+    if (line) {
+      addText(line, fontSize, fontType);
+    }
+    yPosition -= 10; // Extra spacing after paragraph
+  };
+  
+  addText('EMPLOYMENT AGREEMENT', 18, boldFont);
+  yPosition -= 10;
+  
+  addParagraph(`This Employment Agreement ("Agreement") is entered into on ${data.startDate || '[DATE]'} between ${data.companyName || 'Vargas JR'} ("Company") and ${data.employeeName || '[EMPLOYEE NAME]'} ("Employee").`);
+  
+  addText('1. POSITION AND DUTIES', 14, boldFont);
+  addParagraph(`Employee is hereby employed as ${data.position || '[POSITION TITLE]'}. Employee agrees to perform such duties and responsibilities as may be assigned by the Company and to devote their full business time and attention to the Company's business.`);
```

Comment:
Ok yea we want to use contractor because Vargas JR will _not_ devote its full time and attention to one company